designing-reports/styling-reports/styling-report-items.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Styling Report Items
+page_title: Styling Report Items 
+description: Styling Report Items
```

Comment:
You can consider different description. Maybe something like "The article provides the available properties that can be used for the report items styling."